reviews/views.py

```diff
@@ -1,3 +1,25 @@
-from django.shortcuts import render
+from django.http  import JsonResponse
+from django.views import View
 
-# Create your views here.
+from .models    import Review, ReviewImage
+from core.utils import login_decorator
+
+class ReviewView(View):
+    @login_decorator
+    def delete(self, request, room_id):
+        try:
+            Review.objects.get(id=request.GET.get('review_id'), user=request.user).delete()
```

Comment:
review_id도 path parameter로 받아주세요.
```suggestion
            Review.objects.get(id=review_id, room_id=room_id, user=request.user).delete()
```

Reply:
넵~!!